values.yaml

```diff
@@ -187,6 +187,12 @@ scheduler:
   #   cpu: 100m
   #   memory: 128Mi
 
+  # This setting can overwrite
+  # podMutation settings - be sure
+  # to reference the default if you
+  # are making use of those settings.
+  airflowLocalSettingsOverride: ~
```

Comment:
@sjmiller609 what do you think about shortening to `airflowLocalSettings` and dropping `Override` for consistency. `Override` can become redundant so we've opted to just name things precisely what they are.

Reply:
yes that's better.